queries/cylc/folds.scm

```diff
@@ -0,0 +1,10 @@
+[
+  (multiline_string)
+  (multiline_graph_string)
+] @fold
+
+; any section
+(_
+  brackets_open: _
+  name: _?
+  brackets_close: _) @fold
```

Comment:
explicitly declaring all of the `*_section` will be better

Reply:
This is still not resolved yet. I'd prefer explicitly declaring each section like how you did it in indents.scm

Reply:
Sorry, I'm adding all these changes from VSCode and I resolved the conversation from there, while not having pushed the fixes yet.
I'm still learning the GitHub PR "best practices" workflow :/

Reply:
"Don't use VS Code" :)

Reply:
**opens MS Word instead.*